packages/client-core/src/harmony/components/inviteHarmony/FriendTab.tsx

```diff
@@ -51,7 +51,6 @@ const FriendTab = () => {
                   {el === 'Phone' ? <PhoneIphone className={classes.whiteIcon} /> : ''}
                   {el === 'Email' ? <Email className={classes.whiteIcon} /> : ''}
                   {el === 'Invite Code' ? <AccountCircle className={classes.whiteIcon} /> : ''}
-                  {el === 'Friend' ? <SupervisedUserCircle className={classes.whiteIcon} /> : ''}
```

Comment:
Not sure why this icon was removed. It should be restored, or some other icon used in its place.